src/Workspaces/SharedUtilitiesAndExtensions/Compiler/CSharp/Services/SyntaxFacts/CSharpSyntaxFacts.cs

```diff
@@ -730,6 +730,7 @@ public bool IsMethodLevelMember(SyntaxNode node)
         {
             return node is BaseMethodDeclarationSyntax ||
                 node is BasePropertyDeclarationSyntax ||
+                node is EventDeclarationSyntax ||
```

Comment:
The base type of `EventDeclarationSyntax` is `BasePropertyDeclarationSyntax`, so this can be simplified:

```suggestion
                false ||
```

Or one step further:

```suggestion
```

Reply:
good point :)